docs/WALKTHROUGH.md

```diff
@@ -0,0 +1,394 @@
+    status: "True"
+    type: Ready
+```
+
+Notice that the controller has set this brokers `status` field to reflect that
+it's catalog has been added to our cluster's catalog.
+
+## Step 5 - Viewing ServiceClasses
+
+The controller created a `ServiceClass` for each service that the broker we
+added provides. We can view the `ServiceClass` resources available in the
+cluster by doing:
+
+```console
+kubectl get serviceclasses
+NAME                    KIND
+user-provided-service   ServiceClass.v1alpha1.servicecatalog.k8s.io
+```
+
+The `Broker` resource we created points to our UPS broker, which provides a service
+called the `user-provided-service`.  Run the following command to get detail on this service:
+
+```console
+kubectl get serviceclasses user-provided-service -o yaml
+```
+
+We should see something like:
+
+```yaml
+apiVersion: servicecatalog.k8s.io/v1alpha1
+kind: ServiceClass
+metadata:
+  creationTimestamp: 2017-03-03T04:11:17Z
+  name: user-provided-service
+  resourceVersion: "7"
+  selfLink: /apis/servicecatalog.k8s.io/v1alpha1/serviceclassesuser-provided-service
+  uid: 72fef5ce-ffc7-11e6-b111-0242ac110005
+brokerName: ups-broker
+osbGuid: 4F6E6CF6-FFDD-425F-A2C7-3C9258AD2468
+bindable: false
+planUpdatable: false
+plans:
+- name: default
+  osbFree: true
+  osbGuid: 86064792-7ea2-467b-af93-ac9694d96d52
+```
+
+## Step 6 - Provisioning a new Instance
+
+Now that one or more `ServiceClass` resources are in the catalog, we can provision a new
+instance of the `user-provided-service`. We do this by creating a new 
+[`Instance`](../contrib/examples/walkthrough/ups-instance.yaml) resource for each provision:
+
+```console
+./kubectl16 create -f contrib/examples/walkthrough/ups-instance.yaml
+```
+
+That operation should output:
+
+```console
+instance "ups-instance" created
+```
+
+Now that the new `Instance` is created, we can check the status of it with:
+
+```console
+kubectl get instances -n test-ns ups-instance -o yaml
+```
+
+We should see something like:
+
+```yaml
+apiVersion: servicecatalog.k8s.io/v1alpha1
+kind: Instance
+metadata:
+  creationTimestamp: 2017-03-03T04:26:08Z
+  name: ups-instance
+  namespace: test-ns
+  resourceVersion: "9"
+  selfLink: /apis/servicecatalog.k8s.io/v1alpha1/namespaces/test-ns/instances/ups-instance
+  uid: 8654e626-ffc9-11e6-b111-0242ac110005
+spec:
+  osbGuid: 34c984e1-4626-4574-8a95-9e500d0d48d3
+  planName: default
+  serviceClassName: user-provided-service
+status:
+  conditions:
+  - message: The instance was provisioned successfully
+    reason: ProvisionedSuccessfully
+    status: "True"
+    type: Ready
+```
+
+## Step 7 - Bind to the Instance
+
+Now that our `Instance` has been created, we can bind to it. After the bind operation is
+complete on the UPS broker server, the service catalog will write the resulting credentials 
+to a Kubernetes secret. Here's how we create the new 
+[`Binding`](../contrib/examples/walkthrough/ups-binding.yaml):
+
+
+```console
```

Comment:
Although I know it was mentioned earlier, it might be good to state again that the user needed to have made the test-ns namespace before running this.